packages/domainproperties/src/stories/AssayPropertiesPanel.tsx

```diff
@@ -73,4 +75,13 @@ storiesOf("AssayPropertiesPanel", module)
         return (
             <WrappedAssayPropertiesPanel data={elispotAssaySaved.data}/>
         )
-    });
\ No newline at end of file
+    })
+    .add("GPAT Error", () => {
```

Comment:
Was this something you were still working on? I am not seeing any error related UI when I pull this up in storybook.